test/chpldoc/compflags/version/projectVersion.error.doc.good

```diff
@@ -0,0 +1 @@
+error: Invalid version format: 1.0.3-one
```

Comment:
Does this pass start_test?  I'm pretty sure start_test output always ends on the line following the last bit of text